config/default.schema.json

```diff
@@ -1175,6 +1175,16 @@
 			"default": false,
 			"description": "Toggle for the IMSCC course download feature."
 		},
+		"FEATURE_COMMON_CARTRIDGE_COURSE_IMPORT_ENABLED": {
+			"type": "boolean",
+			"default": false,
+			"description": "Toggle for the Common Cartridge course import feature."
+		},
+		"FEATURE_COMMON_CARTRIDGE_COURSE_IMPORT_MAX_FILE_SIZE": {
+			"type": "integer",
+			"default": 2000000000,
+			"description": "The maximum file upload size for the Common Cartridge file during import."
```

Comment:
for easy maintability, you can add the unit in description, e.h maximum file size in bytes
in fact, it would be nicer to have a readable number with another unit.

Reply:
I updated the description. The library we are using for the file upload requires a number in bytes. Another unit would mean random multipliers in the code base and this would not be good for maintainability.

Reply:
Done.